### PR TITLE
Rearranges the Science Department on IceBox Station for Toxins Gas Storage.

### PR DESCRIPTION
**Pull request by san7890**
## About The Pull Request
This map change seeks to resolve issues with IceBox Station that have been present for a very, very, long time.

On the maps currently in rotation, Toxins Gas Storage is connected to the actual Toxins mixing laboratory. This is true for all of them, except IceBox Station. Toxins is extremely discouraging to do when you have to waste precious time running outside of your lab to get your cans of Oxygen and Plasma. It makes no sense for IceBox to be so radically different than Meta or Delta Station's Toxins, where there is usually a hallway facing door and a door that connects right to the mixing lab. Also, for some reason, IceBox has always just had a shitload of empty t1 canisters and portable scrubbers, which were extremely useless in the scope of Toxins, and probably any other aspect in the game.

This PR seeks to accomplish this change by swapping around the RD's office with Toxin's storage. This does shrink the RD's office a teeny bit, but everything was carried over. In order to reduce the amount of empty space in the Toxins Mixing Lab, I decided to expand Genetics' Monkey Pen, and actually add grass. (i don't know how genetics work please tell me if this is a powercreep so i can just reduce the monkey spawns or something) It also does reduce a bit of maintenance as well, but i think it's not as clunky as it previously was, and feels a bit smoother.

## Why It's Good For The Game

As stated above, IceBox Science is so much more different than the other stations, and for no real good reason at all (than what I can think of). This PR reworks that part of Science in order to make it more in form with the other stations currently in rotation, and not have trained bomb-makers pull out their hair when people vote for IceBox.

This PR also doesn't add any additional gas canisters to Toxins (in comparison to what was already there), it just shifts it to another place.

## Changelog
:cl:
add: Bombmakers unite! Nanotrasen has recently retrofitted the Science department on the IceBox Station and made the Toxins Gas Storage room adjacent to... Toxins! They had to sell all of those scrubbers and gas canisters to pay for it though.
del: The RD however, is not too jazzed about his new office on IceBox, It's cut down by an incredibly slim margin, but...... What's the upside again?
add: Genetics Monkey Pen has shifted... Odd...
/:cl:

Before:
image

After:
image

Area Overlay:
image

(this is my first PR go easy on me)

### CODE REVIEW

**Comment by wesoda25** [comment]
I kinda always liked that little room off to the side with all the canisters, it was something seemingly unique. Guess that’s not really an argument though so 👍

**Comment by tralezab** [comment]
Naw wesoda that's a bit valid, there is something so fucking fun about turning the canister closet into the hell box

**Comment by san7890** [comment]
> Naw wesoda that's a bit valid, there is something so fucking fun about turning the canister closet into the hell box

you can still make a wacky ass hellbox in those canister room, it just has two doors now

**Comment by Hamcha** [comment]
I love the strategic placement of the O2/Plasma canisters!

My only gripe: I kinda feel bad that a lot of space translated into that huge monkey pen, does gene even need that many monkeys at roundstart?

**Comment by san7890** [comment]
> I love the strategic placement of the O2/Plasma canisters!
> 
> 
> 
> My only gripe: I kinda feel bad that a lot of space translated into that huge monkey pen, does gene even need that many monkeys at roundstart?

I tried to expand maints, but it all just looked bad. I ended up committing to the namesake of Ice Box, and just boxed out all the rooms. I think Meta has the same amount of roundstart monkeys, correct me if I'm wrong on that.

**Comment by san7890** [comment]
AppVeyor appears to have had a stroke, it compiles fine as of the last commit.

**Comment by Tlaltecuhtli** [comment]
but a fire extinguisher cabinet inside the toxin storage

**Comment by san7890** [comment]
> but a fire extinguisher cabinet inside the toxin storage

The original Toxins storage didn't have one in the room, but rather right outside the door. I replicated this by having either entrance to the room have a fire extinguisher.

**Comment by Fikou** [comment]
the monky pen is ugly
the rd office is really weird now, just random computers, no chair or anything, and its missing a wall
the canisters are kinda weirdly placed?

**Comment by san7890** [comment]
> the monky pen is ugly

How would you recommend I fix that?
> the rd office is really weird now, just random computers, no chair or anything, and its missing a wall

I forgot to add the chair, my bad. What wall is it missing? I'm not at my computer so I'm just looking at the renders. I had a different setup for the computers that fit a lot better, but the whole thing involving the RD's facehugger just fucks with the whole flow. I feel like a cramped, stuffy office, encapsulates the essence of the RD though. A geeky nerd in a beige suit wants an office cramped full of all his geeky shit, with computers and miscellaneous items strewn about.
> the canisters are kinda weirdly placed?

I might remove a singular can of plasma to make the co2 canister fit in right. Four is plenty.

I'm away from my computer right now, but I'll look over and incorporate stuff I've fluked tommorow.

**Comment by Fikou** [comment]
just keep the monkey pen the same as it was before and enlargen the room by 2 and fill it with some more canisters and scrubbers

**Comment by san7890** [comment]
Sounds good but:
> fill it with some more canisters and scrubbers
image

I've got an idea on how to handle it though.
> the wall in rd's is missing in the window part

I have no fucking idea what causes this. The map render on the latest commit says it's fine (not missing). Is there an /obj/structure/blob/core on the tile where the computer is? For some reason, that kept coming back up and removing a window. I thought I had fixed it, and I hope it's not a code issue.

**Comment by san7890** [comment]
alright, i figured it all out today and pushed the changes here. the monkey pen isn't "weird" anymore, and I've corrected any other flaws. please let me know if you need any changes done.

**Comment by san7890** [comment]
@ShizCalev is there anything else that needs to be done?

**Review by Timberpoes — APPROVED** [review]
Yo! Sorry for the radio silence. You've not been forgotten, or if you were you now are not.

Tentatively approving these changes. 12-24 hour delay before I merge incase any pressing issues crop up. If any maintainer wants to block this, I'm happy to cede.

The reason I'm only tentatively approving is because our stations are different and are meant to be different. If everything had the same layout, then why do we have different stations at all?

However, change is sometimes good. Changes like this mix things up, break familiarity and can make people feel a bit uneasy, which is a pretty cool feeling to have in a paranoia game like SS13.

So I'm going this a classic Timberpoes Thumbs Neither Up Nor Down and will look to merge this as per my above timescale.

**Comment by san7890** [comment]
> Yo! Sorry for the radio silence. You've not been forgotten, or if you were you now are not.
> 
> Tentatively approving these changes. 12-24 hour delay before I merge incase any pressing issues crop up. If any maintainer wants to block this, I'm happy to cede.
> 
> The reason I'm only tentatively approving is because our stations are different and are meant to be different. If everything had the same layout, then why do we have different stations at all?
> 
> However, change is sometimes good. Changes like this mix things up, break familiarity and can make people feel a bit uneasy, which is a pretty cool feeling to have in a paranoia game like SS13.
> 
> So I'm going this a classic Timberpoes Thumbs Neither Up Nor Down and will look to merge this as per my above timescale.

thanks for the reply. I get your point that diversity in maps makes them each uniquely different, but Box Science is already much more different than that of Meta or Delta. This change only addresses the fact that one of the key components to a specialty of Science is unnecessarily further away from the actual workplace, and is not conducive at all. It's like if the Genetics monkey pen was a 15 second walk away, and in another section of science than where the DNA consoles are. It discourages the level of innovation that Toxins can achieve already (50k points, anomaly cores) by adding an unnecessary complexity. 

I'll happily take the late-merge over any sort of "pocket close", though.

**Comment by Timberpoes** [comment]
Looks like I got pre-empted! Looks like Jared liked it enough to merge it. Your lucky day!

**Pull request merged**
